test/odbc/psqlodbc/test_pg_datatype_real.cpp

```diff
@@ -0,0 +1,933 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include "../src/drivers.h"
+#include <cmath>
+#include <iostream>
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.real_table_odbc_test";
+const string COL1_NAME = "pk";
+const string COL2_NAME = "data";
+const string DATATYPE_NAME = "sys.real";
+const string VIEW_NAME = "master_dbo.real_view_odbc_test";
+const vector<pair<string, string>> TABLE_COLUMNS = {
+  {COL1_NAME, "INT PRIMARY KEY"},
```

Comment:
Why not `real` for primary key?

Reply:
Consistency with other tests.
The Primary key in most tests are inserted using a for loop's index and with some tests, using the test type wouldn't work eg DateTime